content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -149,30 +149,30 @@ the version.
 When an object is written, it is persisted at the version designated as the
 storage version at the time of the write. If the storage version changes,
 existing objects are never converted automatically. However, newly-created
-or updated objects are created at the new storage version. It is possible for an
+or updated objects are written at the new storage version. It is possible for an
 object to have been written at a version that is no longer served.
 
 When you read an object, you specify the version as part of the path. If you
 specify a version that is different from the object's persisted version,
-Kubernetes returns the object to you at the version you requested, but does not
-modify the persisted object. You can request an object at any version that is
-currently served.
+Kubernetes returns the object to you at the version you requested, but
+the persisted object is not changed on disk and is returned without any conversion.
```

Comment:
but the persisted object is neither changed on disk, nor converted in any way (other than changing the `apiVersion` string) while serving the request.

Reply:
Done.